src/hammer-vlsi/technology/asap7/sram-cache.json

```diff
@@ -1,10 +1,415 @@
 [
+{
+  "type" : "sram",
```

Comment:
If these all have the same ports do we have the ability to use the more compact sram-compiler syntax?

Reply:
Hm, I will need to examine, but I was under the impression that the ranges syntax was only available for MacroCompiler's --user-compiler mode.

Reply:
Verdict is that yes, it would work. However, given the selection of SRAMs I have (in `srams.txt`), it would save <10% of the json entries. Also would complicate the `sram-cache-gen.py` script used to generate new versions of the `sram-cache.json` file.